packages/ts-migrate/cli.ts

```diff
@@ -84,7 +86,16 @@ yargs
           process.exit(1);
           return;
         }
-        config = new MigrateConfig().addPlugin(plugin, {});
+        const options: any = {};
```

Comment:
This all feels a little hacky, so happy to make changes if there are any thoughts on the "right" way to do this.

Reply:
maybe you can define an options type somewhere here https://github.com/airbnb/ts-migrate/blob/master/packages/ts-migrate-server/types/index.ts and use it here?

Reply:
I'm quite sure what you mean. I've already defined an options type alongside the plugin, but it's not exported. I could export it, but that's in a different package than the file you've linked to.

I've tried a different approach which creates a little bit of duplication but means that TypeScript is able to infer the correct type for the options parameter.